x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/actions/SearchableSnapshotActionIT.java

```diff
@@ -188,4 +196,149 @@ public void testDeleteActionDeletesSearchableSnapshot() throws Exception {
         }, 30, TimeUnit.SECONDS));
     }
 
+    public void testCreateInvalidPolicy() throws Exception {
+        String snapshotRepo = randomAlphaOfLengthBetween(4, 10);
+        createSnapshotRepo(client(), snapshotRepo, randomBoolean());
+        IllegalArgumentException exception = expectThrows(IllegalArgumentException.class, () -> createPolicy(client(), policy,
+            new Phase("hot", TimeValue.ZERO, Map.of(SearchableSnapshotAction.NAME, new SearchableSnapshotAction(snapshotRepo))),
+            new Phase("warm", TimeValue.ZERO, Map.of(ForceMergeAction.NAME, new ForceMergeAction(1, null))),
+            new Phase("cold", TimeValue.ZERO, Map.of(FreezeAction.NAME, new FreezeAction())),
+            null
+            )
+        );
+
+        assertThat(exception.getMessage(), is("phases [warm,cold] define one or more of [shrink,forcemerge,freeze] actions which are not " +
+            "allowed after the managed index was mounted as searchable snapshot"));
+    }
+
+    public void testUpdatePolicyToAddPhasesYieldsInvalidActionsToBeSkipped() throws Exception {
```

Comment:
@dakrone do you think we should aim to prevent this case when updating the policy?

Reply:
No I don't think so, I think we should allow a user to remove searchable snapshots from the policy as well as adding other actions (post searchable snapshot for instance) that should run just fine.